### PR TITLE
Filtern

### PR DESCRIPTION
**Pull request by AnastasiyaVasileuskaya**

### CODE REVIEW

**Comment by laurastrehlow** [comment]
Very nice work!

**Review by laurastrehlow — APPROVED** [review]
Very nice work!

File: src/components/App.js [hunk 1]
```diff
@@ -2,47 +2,108 @@ import { useEffect, useState } from 'react'
...
 
+  useEffect(() => {
+    setUrl(creatUrlQuery())
+  }, [query, caloriesRangeFrom, caloriesRangeTo, healthLabels, dishTypes])
+
+  useEffect(() => {
+    getRecipes()
+  }, [url])
+
+  function creatUrlQuery() {
```
Comment (BerePendzialek) [inline src/components/App.js:49]: Hi, would it be possible to save this function outside this file?

Comment (AnastasiyaVasileuskaya) [reply src/components/App.js:49]: Hi,good idea!Thanks for your suggestion!

**Pull request merged**